sample/src/main/res/values/dimens.xml

```diff
@@ -2,4 +2,8 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="map_marker_min_width">34dp</dimen>
+    <dimen name="map_marker_min_height">30dp</dimen>
+    <dimen name="map_marker_text_size">15dp</dimen>
```

Comment:
all of these things seem like they come from the airbnb repo. maybe we should just have a simple bitmap marker for the sample from a resource